templates/home/_fsf_python.html

```diff
@@ -16,7 +16,7 @@ <h4>Why are snaps good for Python projects?</h4>
   </div>
 
   <div class='col-6'>
-    <h4>How <a href="https://github.com/OfflineIMAP/offlineimap">offlineimap</a> defines <code>snapcraft.yaml</code></h4>
+    <h4>How <a href="/offlineimap">offlineimap</a> defines <code><a href="https://github.com/OfflineIMAP/offlineimap/blob/master/snapcraft.yaml" target="_blank">./snapcraft.yaml</a></code></h4>
```

Comment:
I think this comes from another PR, you need to rebase on master once it's merged